Watch Extension/HomeView.swift

```diff
@@ -0,0 +1,45 @@
+import SwiftUI
+
+/*
+ TODO - V1 Watch
+ - Polish UI
+ - Make running feedings a live list
+ - Make sure a backgrounded/terminated main app
+   behaves well from a UI standpoint when using the watch
+ - App icon
+ - Think about subscriptions or IAP for this
+ - Better formalize communication instead of basing it off
+   of yucky strings
+ - Add unidirectional flow architecture for better state management
+ */
+
+struct HomeView: View {
+    @State private var isShowingSheet = false
+    var body: some View {
+        VStack {
+            // TODO: fill this in with active feedings
+            List((1...10), id: \.self) { num in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/c49e444d61917df61aa42d9cccb11a46857e6fb9/Watch Extension/HomeView.swift#L21"></span>Unused parameter “num” in a closure should be replaced with _.<br />
<code>unused_closure_parameter</code> <code>HomeView.swift:21</code></td>
    </tr>
  </tbody>
</table>